arango/request.py

```diff
@@ -2,13 +2,21 @@
 
 from typing import Any, MutableMapping, Optional
 
-from arango.typings import Fields, Headers, Params
+from arango.typings import DriverFlags, Fields, Headers, Params
 
 
-def normalize_headers(headers: Optional[Headers]) -> Headers:
+def normalize_headers(
+    headers: Optional[Headers], driverFlags: Optional[DriverFlags] = None
+) -> Headers:
+    flags = ""
+    if driverFlags is not None:
+        for flag in driverFlags:
+            flags = flags + flag + ";"
+    driverInfo = "python-arango/7.5.2 (" + flags + ")"
```

Comment:
Same as above. `driverInfo` -> `driver_info` (or better yet `driver_header`).

Reply:
This is fixed